src/utils/web3.js

```diff
@@ -48,13 +52,22 @@ const BLOCK_EXPLORERS = {
     const typePart = ETHERSCAN_TYPES.get(type)
     return `https://${subdomain}etherscan.io/${typePart}/${value}`
   },
-  blockscout: ({ type, value }) => {
+  blockscout: ({ type, value, networkType }) => {
+    if (networkType === 'private') {
+      return ''
+    }
+
+    if (!BLOCKSCOUT_NETWORK_TYPES.has(networkType)) {
+      throw new Error('provider not supported.')
+    }
+
     if (!BLOCKSCOUT_TYPES.has(type)) {
       throw new Error('type not supported.')
     }
 
-    const typePart = ETHERSCAN_TYPES.get(type)
-    return `https://blockscout.com/xdai/mainnet/${typePart}/${value}`
+    const networkName = ETHERSCAN_NETWORK_TYPES.get(networkType)
```

Comment:
I think we made the same mistake 😄 
```suggestion
    const networkName = BLOCKSCOUT_NETWORK_TYPES.get(networkType)
```